DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Repositories/AcademiesDbDataSourceRepository.cs

```diff
@@ -0,0 +1,59 @@
+using DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Contexts;
+using DfE.FindInformationAcademiesTrusts.Data.Enums;
+using DfE.FindInformationAcademiesTrusts.Data.Repositories;
+using DfE.FindInformationAcademiesTrusts.Data.Repositories.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Repositories;
+
+public class AcademiesDbDataSourceRepository(
```

Comment:
Can we remove the prefix AcademiesDb from the class name, we don't use it for the other repos, I think being in the academies db project should be all good for conveying that.

Reply:
Done